skimage/transform/hough_transform.py

```diff
@@ -63,7 +62,7 @@ def hough_line_peaks(hspace, angles, dists, min_distance=9, min_angle=10,
                                min_ydistance=min_distance,
                                threshold=threshold,
                                num_peaks=num_peaks)
-    if a.any():
+    if a.size > 0:
```

Comment:
This is the line fixes the bug. the other modified lines are flake8/isort fixes and the new test case